masterlist.yaml

```
@@ -4982,6 +4982,21 @@ plugins:
       - crc: 0x3291418D
         util: 'TES4Edit v4.0.4b'
 
+  - name: 'OCOv2 - (Filter|MOO) Patch( for Mods)?\.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/51379/'
+        name: 'OCOv2 - Filter Patch for Mods'
+  - name: 'OCOv2 - Filter Patch for Mods.esp'
+    group: *latePatchesGroup
+    msg:
+      - <<: *patchProvided
+        subs: [ 'Maskar''s Oblivion Overhaul' ]
+        condition: 'active("Maskar''s Oblivion Overhaul.esp") and not file("OCOv2 - MOO Patch.esp")'
+  - name: 'OCOv2 - MOO Patch.esp'
```

You could add a message to ensure the user has a Bashed Patch when using this patch:

```suggestion
        condition: 'active("Maskar''s Oblivion Overhaul.esp") and not file("OCOv2 - MOO Patch.esp")'
      - *requiresBashedPatch
  - name: 'OCOv2 - MOO Patch.esp'
```